pkg/services/internal/whatsonchain/get_status_for_txids.go

```diff
@@ -0,0 +1,22 @@
+package whatsonchain
+
+import (
+	"context"
+
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/wdk"
+)
+
+func (woc *WhatsOnChain) getStatusForTxids(ctx context.Context, url string, txids []string) (*wdk.GetStatusForTxidsResult, error) {
+	response, err := woc.doStatusRequest(ctx, url, txids)
+	if err != nil {
+		return nil, err
+	}
+
+	results := mapWocStatusResponse(response)
```

Comment:
Let's try to stick to a single method of mapping slices.
```suggestion
	results := slices.Map(response, mapSingleTxStatus)
```